indico/modules/events/persons/templates/email_dialog.html

```diff
@@ -0,0 +1,19 @@
+{% from 'forms/_form.html' import simple_form %}
+{{ simple_form(form, submit=_('Send')) }}
+<script>
+    (function() {
+        'use strict';
+
+        var $recipients = $('#form-group-recipients').find('.form-field');
+        var count = {{ emails_count }};
```

Comment:
It's super awful to mix javascript code with Jinja except for variable definitions...